std/process.d

```diff
@@ -1280,7 +1291,12 @@ private Pid spawnProcessImpl(scope const(char)[] commandLine,
     if (!CreateProcessW(null, commandLine.tempCStringW().buffPtr,
                         null, null, true, dwCreationFlags,
                         envz, workDir.length ? pworkDir : null, &startinfo, &pi))
-        throw ProcessException.newFromLastError("Failed to spawn new process");
+    {
+        if (GetLastError() == ERROR_FILE_NOT_FOUND)
+            throw new ProcessException(text("Executable file not found: ", program));
```

Comment:
I think the other PR was still a little better in this regard for a few reasons:

1. The error is "**file** not found", not "**executable** not found"
2. We forego error message localization provided by the OS for this error code only
3. It seems inconsistent to include the program name only for this error code, and not others.

Reply:
Well, the other PR was abandoned, resulting in no progress at all. May I suggest you follow up on your own, given that you have a strong opinion about this and seem to care about the improbable error cases and Posix filenames as well? I'm thankful for concerns regarding the DLLs, those haven't crossed my mind at all, but seem to be of no concern.

Reply:
No problem. I don't really have a strong opinion on this, just pointing out a few opportunities for improvement while we're on the subject.